package.json

```diff
@@ -41,6 +41,7 @@
   },
   "scripts": {
     "karma": "karma start vision/explore-api/karma.conf.js",
+    "karma": "karma start speech/explore-api/karma.conf.js",
```

Comment:
Note I ended up splitting the commands so that the failures of the first sample did not short-circuit the second set of tests from running.